examples/unconditional_image_generation/train_unconditional.py

```diff
@@ -59,6 +59,11 @@ def main(args):
             "UpBlock2D",
         ),
     )
+
+    if args.ort:
```

Comment:
This is a pytorch only training example, let's please not add onnxruntime here. The premise of the training scripts is to be **very** easy to run and understand. People should not have to know onnxruntime IMO to run the simplest training example that we have